cypress/support/step-definitions/radio-button-steps.js

```diff
@@ -17,14 +22,230 @@ Then('fieldHelpInline is disabled', () => {
 When('I click onto {string} radioButton for validations component into iFrame', (position) => {
   switch (position) {
     case 'first':
-      radioButton().eq(FIRST_ELEMENT).click();
+      radioButton().eq(FIRST_RADIOBUTTON).click();
       break;
     case 'second':
-      radioButton().eq(SECOND_ELEMENT).click();
+      radioButton().eq(SECOND_RADIOBUTTON).click();
       break;
     case 'third':
-      radioButton().eq(THIRD_ELEMENT).click();
+      radioButton().eq(THIRD_RADIOBUTTON).click();
       break;
     default: throw new Error('There are only three validation icon elements on the page');
   }
 });
+
+Then('{string} radioButton on preview is {string}', (position, text) => {
+  switch (position) {
+    case 'First':
+      labelByPosition(FIRST_RADIOBUTTON).should('have.text', text);
+      break;
+    case 'Second':
+      labelByPosition(SECOND_RADIOBUTTON).should('have.text', text);
+      break;
+    case 'Third':
+      labelByPosition(THIRD_RADIOBUTTON).should('have.text', text);
+      break;
+    default: throw new Error('There are only three radio button elements on the page');
+  }
+});
+
+Then('{string} RadioButton has value {string}', (position, text) => {
+  switch (position) {
+    case 'First':
+      radioButtonByPosition(FIRST_RADIOBUTTON).should('have.attr', 'value').should('contain', text);
+      break;
+    case 'Second':
+      radioButtonByPosition(SECOND_RADIOBUTTON).should('have.attr', 'value').should('contain', text);
+      break;
+    case 'Third':
+      radioButtonByPosition(THIRD_RADIOBUTTON).should('have.attr', 'value').should('contain', text);
+      break;
+    default: throw new Error('There are only three radio elements on the page');
+  }
+});
+
+Then('{string} RadioButton component is disabled', (position) => {
+  switch (position) {
+    case 'First':
+      radioButtonComponent(FIRST_RADIOBUTTON).should('have.attr', 'disabled');
+      radioButtonByPosition(FIRST_RADIOBUTTON).should('have.attr', 'disabled');
```

Comment:
we should add also `.should('be.disabled')` to check :)

Reply:
fixed